scripts/update-terraform-modules.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+## Init temp dir
+tmp_dir=tmp
+mkdir -p $tmp_dir
+
+github_org=cloudposse
+github_api_limit=100
+github_api_pages=3
+
+modules=[]
+
+## Fetch repository names
+echo "Fetching repository names..."
+if [ ! -f $tmp_dir/modules_cache.txt ]; then
+    modules=$(
+        seq $github_api_pages | \
+        xargs -I "{}" curl -sSL "https://api.github.com/orgs/${github_org}/repos?per_page=${github_api_limit}&page={}" | \
+        jq -r '.[] | .name' | \
+        sort | grep terraform-*
+    )
+    echo "$modules" > $tmp_dir/modules_cache.txt
+else
+    modules=$(<$tmp_dir/modules_cache.txt)
+fi
+
+## Clone repositories
+echo "Cloning repositories..."
+mkdir -p $tmp_dir/projects/terraform
+for module in $modules
+do
+    module_dir=$tmp_dir/projects/terraform/$module
+    if [ ! -d $module_dir ]; then
+        git clone https://github.com/$github_org/$module.git $module_dir
```

Comment:
I don't understand why we need to clone anything.

Reply:
We need to clone terrafrom repos because we need files from it.
The README.yaml is needed (and it's stong dependency), but there are also .md files (includes) in `docs` dir in most of terraform repositories.

Reply:
I can propose to categorize it in following way:
E.g. `terraform-aws-ecs-cloudwatch-autoscaling.md`
```
terraform-modules/
  aws/ (first group)
    ecs/ (second group)
      cloudwatch/ (third group)
        autoscaling.md (the page)
```

But there should be exact name pattern for each terraform-module.

Reply:
I'm not sure... I like that it consistent and automated. But what I don't like is that it's a lot of depth. That doesn't look good in our site navigation menu. Also, it's not the way I want to organize it for users. 

What I'm thinking is the user runs `mkdir` in `terraform-modules` somewhere. Our script just looks for those directories and attempts to generate the documentation.

Reply:
> Our script just looks for those directories and attempts to generate the documentation.

but how to map terraform-module-pages to user folder?